### PR TITLE
mariadb@10.1 10.1.24

### PR DESCRIPTION
**Pull request by makigumo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/mariadb@10.1.rb [hunk 1]
```diff
@@ -24,7 +24,6 @@ class MariadbAT101 < Formula
   deprecated_option "with-tests" => "with-test"
 
   depends_on "cmake" => :build
-  depends_on "pidof" unless MacOS.version >= :mountain_lion
```
Comment (ilovezfs) [inline Formula/mariadb@10.1.rb]: why is this being removed?

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/mariadb@10.1.rb (hunk 1 above)
Comment (makigumo) [inline Formula/mariadb@10.1.rb]: Sharp eyes 👍 
Anyway it was a leftover, `pidof` is no longer used. And if I read https://github.com/Homebrew/homebrew-core/commit/7c6dc47bf261616e75542d8ff10ec1326e31ffe3#diff-16c08a700c21562997855c92d2787d15L118 correctly for some time already.

Comment (ilovezfs) [inline Formula/mariadb@10.1.rb]: That change is `if MacOS.version >= :mountain_lion` which means pidof would still be need for < :mountain_lion unless I misunderstood something.

Comment (makigumo) [inline Formula/mariadb@10.1.rb]: You are right! But as already mentioned `pidof` is no longer required, they use `pgrep` now.

Comment (ilovezfs) [inline Formula/mariadb@10.1.rb]: Ah OK.

**Review by ilovezfs — APPROVED** [review]

**Review by JCount — APPROVED** [review]

**Comment by JCount** [comment]
@makigumo Homebrew appreciates your contributions!! 💯

**Pull request closed without merging**